# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1437705334 14400
#      Thu Jul 23 22:35:34 2015 -0400
# Node ID c16c446204e5564d26051f8884f39ba5e55ed9d0
# Parent  956119b6e6d14cdc59bec8094985f71dd083fe99
maint: Update .hgignore to reflect non-recursive build system changes

* .hgignore: Update ignore patterns for generated Texinfo files.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -27,16 +27,16 @@ syntax: regexp
 ^configure$
 ^autom4te\.cache($|/)
 ^config\.in\.h$
 
 # e.g. doc/faq/OctaveFAQ.info
 #      doc/interpreter/octave.info-4
 ^doc/.*\.info(-\d)?$
 
-^doc/\w*/stamp-vti$
-^doc/\w*/version\.texi$
+^doc/\w*/stamp-(vti|\d+)$
+^doc/\w*/version-\w+\.texi$
 ^doc/interpreter/images\.mk$
 
 # e.g. liboctave/operators/smx-op-inc.mk
 ^liboctave/operators/\w+-op-\w+\.mk$
 
 ^libinterp/dldfcn/module\.mk$
